src/blocknative.rs

```diff
@@ -0,0 +1,180 @@
+use super::{linear_interpolation, GasPriceEstimating, Transport};
+use anyhow::{anyhow, Context, Result};
+use serde::Deserialize;
+use std::{convert::TryInto, time::Duration};
+
+// Gas price estimation with https://www.blocknative.com/gas-estimator , api https://docs.blocknative.com/gas-platform#example-request .
+
+const API_URI: &str = "https://api.blocknative.com/gasprices/blockprices";
+
+const PERCENT_99: Duration = Duration::from_secs(15);
```

Comment:
We could also calculate this on the fly based on the confidence level in the response. They don't explicitly say the default confidence levels can change but if it doesn't get more complex we might as well. In this case it means the hardcoded list of 5 becomes a loop and we'd collect into a vec which I think is fine.

Reply:
Fixed.